### ISSUE TITLE
stepfunctions_tasks: Upgrade the volume type when creating EMR cluster with the EmrCreateCluster task

### ISSUE THREAD

**Issue by lafeuil** [aws/aws-cdk#28874]
### Describe the bug

In the step function, when creating an EMR cluster with the [EmrCreateCluster](https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_stepfunctions_tasks.EmrCreateCluster.html) task, the EBS volume of each instance can be configured. The volume is described by the [VolumeSpecificationProperty](https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_stepfunctions_tasks.EmrCreateCluster.VolumeSpecificationProperty.html) interface and the type of each volume by the [volumeType](https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_stepfunctions_tasks.EmrCreateCluster.VolumeSpecificationProperty.html#volumetype) property and the [EbsBlockDeviceVolumeType](https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_stepfunctions_tasks.EmrCreateCluster.EbsBlockDeviceVolumeType.html) enum.

But in the `EbsBlockDeviceVolumeType`, there are only the `gp2`, `io1` and `standard` type. The EMR specification of the `RunJobFlow` action accepts for the [VolumeType](https://docs.aws.amazon.com/emr/latest/APIReference/API_VolumeSpecification.html#EMR-Type-VolumeSpecification-VolumeType) property : `gp3`, `gp2`, `io1`, `st1`, `sc1`, and `standard`.

### Expected Behavior

The [EmrCreateCluster](https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_stepfunctions_tasks.EmrCreateCluster.html) task should accept all supported volume type of the `RunJobFlow` action :  `gp3`, `gp2`, `io1`, `st1`, `sc1`, and `standard`.

### Current Behavior

The [EmrCreateCluster](https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_stepfunctions_tasks.EmrCreateCluster.html) task only accepts `gp2`, `io1` and `standard` EBS volume type.

### Reproduction Steps

```typescript
new EmrCreateCluster(scope, 'Create Cluster', {
  name: 'MyCluster',
  instances: {
    instanceFleets: [
      {
        instanceFleetType: EmrCreateCluster.InstanceRoleType.CORE,
        instanceTypeConfigs: [
          {
            instanceType: 'r6g.xlarge',
            ebsConfiguration: {
              ebsBlockDeviceConfigs: [
                {
                  volumeSpecification: {
                    volumeSize: Size.gibibytes(256),
                    volumeType: EmrCreateCluster.EbsBlockDeviceVolumeType.GP3,
                  },
                },
              ],
            },
          },
        ],
      },
    ],
  },
});
```

### Possible Solution

Upgrade the [EbsBlockDeviceVolumeType](https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_stepfunctions_tasks.EmrCreateCluster.EbsBlockDeviceVolumeType.html) enum with the missing value : `gp3`, `st1`, and `sc1`.

### CDK CLI Version

2.123.0

### Node.js Version

20

### OS

linux

### Language

TypeScript

**Comment by pahud** [comment]
Yes we can submit a tiny PR to add all supported types in the enum class:

https://github.com/aws/aws-cdk/blob/6a7a24afcc1ebebf71c267b890732a455e865cc8/packages/aws-cdk-lib/aws-stepfunctions-tasks/lib/emr/emr-create-cluster.ts#L488-L501
